.circleci/config.yml

```diff
@@ -35,34 +35,24 @@ jobs:
             - v1-{{ .Branch }}
           paths:
             - /caches/app.tar
-      - run:
-          name: Load Docker image layer cache
-          command: |
-            set +o pipefail
-            docker load -i /caches/app.tar | true
-      - run:
-          name: Build application Docker image
-          command: |
-            docker build --cache-from=app -t app .
-      - run:
-          name: Save Docker image layer cache
-          command: |
-            mkdir -p /caches
-            docker save -o /caches/app.tar app
-      - save_cache:
-          key: v1-{{ .Branch }}-{{ epoch }}
-          paths:
-            - /caches/app.tar
       - run:
           name: Run docker tests
           command: |
             docker-compose up --abort-on-container-exit --quiet-pull --no-color --exit-code-from=app
+      - run:
+          name: Docker Build
+          command:
+            docker build -t refapp .
       - deploy:
           name: Push application Docker image
           command: |
-            #if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              login="$(aws ecr get-login)"
-              ${login}
-              docker tag app "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}2:${CIRCLE_PROJECT_REPONAME}${CIRCLE_SHA1}"
-              docker push "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}2:${CIRCLE_PROJECT_REPONAME}${CIRCLE_SHA1}"
-            #fi
+            login="$(aws ecr get-login)"
+            ${login}
+
+            docker tag refapp "${ECR_ENDPOINT}/cloud-platform/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1}"
```

Comment:
I think `cloud-platform` here represents the github team name? If so it'd be good to either define this as an environment variable that can be populated on circleci.com, or use our team name (`webops`), and have a comment to explain that it should be changed to the relevant team for the user.

Reply:
Are you sure it should be webops? I thought the team has been renamed into cloud-platform.

Reply:
No, not on Github - it's still `WebOps` there - but when using team names in our code we should always use the lowercased slug version, so `webops` here.